_posts/2018-06-29-14-04-netcore-7-0.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+We're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
+ASP.NET Web API OData 7.0 is available with two new packages:
+
+1. [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) is based on ASP.NET Core 2.0.
+2. [Microsoft.AspNet.OData](https://www.nuget.org/packages/Microsoft.AspNet.OData) is based on ASP.NET Web API.
+
+[Here](http://odata.github.io/WebApi/#14-01-netcore-beta1) can find beta1, beta2 & beta4 information.
+
+[Get started](https://github.com/OData/ODataSamples/tree/master/WebApi/v7.x) with ASP.NET Web API OData 7.0 today!
+
+## Download this release
+
+You can install or update the NuGet packages for OData Web API v7.0.0 using the [Package Manager Console](http://docs.nuget.org/docs/start-here/using-the-package-manager-console):
+
+```
+PM> Install-Package Microsoft.AspNetCore.OData
+```
+or
+```
+PM> Install-Package Microsoft.AspNet.OData
+```
+
+## What’s in this release?
+
+<h4>ASP.NET Core 2.x support</h4>
+
+ See main issues from:
+
+ * [ [Web API issue #939](https://github.com/OData/WebApi/issues/939) ] .NET Core Support - IN Progress
```

Comment:
> - IN Progress [](start = 89, length = 14)

Can we remove the "IN Progress"? Makes it sound like .NET Core is not fully supported in this release.

Reply:
#Resolved.